src/vtol.py

```diff
@@ -135,21 +137,41 @@ def takeoff(self):
 
         print("Reached target altitude")
 
+    def set_altitude(self, alt):
+        '''Sets altitude of quadcopter using an "alt" parameter'''
+        print("Setting altitude:")
+        destination = LocationGlobalRelative(self.location.global_relative_frame.lat, self.location.global_relative_frame.lon, alt)
+        self.simple_goto(destination)
```

Comment:
Just realized, you should call your go_to method you just made instead. I think this code is a bit redundant